tests/integration/test_interaction.py

```diff
@@ -674,6 +674,44 @@ def test_do_attempt_feedback_is_updated(self):
         self.assertEqual(self._get_feedback().text, expected_feedback)
 
 
+class AssessmentGradingTest(
```

Comment:
@arbrandes any reason this is a new class rather than added to existing `AssessmentInteractionTest`?